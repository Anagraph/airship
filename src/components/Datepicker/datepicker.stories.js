import React from 'react';
import { storiesOf } from '@storybook/react';
import { ThemeProvider } from 'styled-components';
import Datepicker from './datepicker';
import { theme } from '../../constants';

const CUSTOM_THEME = {
  ...theme,
  ui01: '#333',
  ui02: '#555',
  type01: 'white',
  type02: 'white',
  type03: 'white',
  brand01: '#FF5500',
};
storiesOf('DatePicker', module)
  .add('Default', () => (
    <Datepicker
      fromMonth={new Date(2018, 8)}
      toMonth={new Date(2020, 11)}
    />
  ))
  .add('With custom theme', () => (
    <ThemeProvider theme={CUSTOM_THEME}>
      <Datepicker
        fromMonth={new Date(2018, 8)}
        toMonth={new Date(2020, 11)}
      />
    </ThemeProvider>
  ));
