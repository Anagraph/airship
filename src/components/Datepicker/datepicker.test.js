import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';
import Datepicker from './datepicker';

describe('render', () => {
  it('renders without crashing', () => {
    const component = renderer.create(
      <Datepicker fromMonth={new Date(2018, 8)} toMonth={new Date(2020, 11)} />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('click day action', () => {
    const spy = jest.fn();
    const component = mount(
      <Datepicker
        onDayClick={spy}
        fromMonth={new Date(2018, 8)}
        toMonth={new Date(2020, 11)}
      />
    );

    // It belongs to Month 7, so it's disabled because it's not in the range of valid dates.
    component
      .find('.DayPicker-Day')
      .at(2)
      .simulate('click');
    expect(spy).not.toHaveBeenCalled();

    component
      .find('.DayPicker-Day')
      .at(10)
      .simulate('click');
    expect(spy).toHaveBeenCalled();
  });
});
