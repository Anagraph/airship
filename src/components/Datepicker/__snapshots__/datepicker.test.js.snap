// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render renders without crashing 1`] = `
.c0 {
  background-color: #fff;
  border-radius: 4px;
  box-shadow: 0 2px 8px 0 rgba(44,44,44,0.16);
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  padding: 8px 16px;
}

.c0 .DayPicker-wrapper {
  border-radius: 4px;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c0 .DayPicker-WeekdaysRow,
.c0 .DayPicker-Week {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c0 .DayPicker-Weekday {
  color: #747474;
  font: 500 10px/24px 'Roboto';
  text-transform: uppercase;
  width: 24px;
  height: 24px;
  text-align: center;
}

.c0 .DayPicker-Weekday + .DayPicker-Weekday {
  margin-left: 4px;
}

.c0 abbr {
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c0 .DayPicker-Day {
  background-color: #F5F5F5;
  color: #1785FB;
  font: 400 12px/24px 'Roboto';
  width: 24px;
  height: 24px;
  outline: none;
  text-align: center;
  cursor: pointer;
}

.c0 .DayPicker-Day + .DayPicker-Day {
  margin-left: 4px;
}

.c0 .DayPicker-Day--outside {
  background-color: transparent;
  color: #BABABA;
}

.c0 .DayPicker-Week + .DayPicker-Week {
  margin-top: 4px;
}

.c0 .DayPicker-Day--selected {
  background-color: #1785FB;
  color: #fff;
  box-shadow: #1785FB 4px 0 0,#1785FB -4px 0 0;
}

.c0 .DayPicker-Day--selected:last-child {
  box-shadow: #1785FB -4px 0 0;
}

.c0 .DayPicker-Day--selected.not(.DayPicker-Day--start):first-child {
  box-shadow: #1785FB 4px 0 0;
}

.c0 .DayPicker-Day--selected.not(.DayPicker-Day--end):last-child {
  box-shadow: #1785FB -4px 0 0;
}

.c0 .DayPicker-Day--start {
  border-radius: 4px 0 0 4px;
  background-color: #035dbd;
  box-shadow: #035dbd 4px 0 0;
}

.c0 .DayPicker-Day--start:last-child {
  box-shadow: none;
}

.c0 .DayPicker-Day--end {
  border-radius: 0 4px 4px 0;
  background-color: #035dbd;
  box-shadow: none;
}

.c0 .DayPicker-Day--start.DayPicker-Day--end {
  border-radius: 4px;
}

.c0 .DayPicker-Day--disabled {
  color: #747474;
  cursor: not-allowed;
}

.c0 .DayPicker-NavBar {
  position: relative;
}

.c0 .DayPicker-NavButton {
  position: absolute;
  cursor: pointer;
  top: 0;
  margin-top: 2px;
  width: 8px;
  height: 16px;
  display: inline-block;
  background-repeat: no-repeat;
  background-position: center;
  outline: none;
}

.c0 .DayPicker-NavButton:hover {
  opacity: 0.8;
}

.c0 .DayPicker-NavButton--prev {
  left: 0;
  background-image: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iOCIgaGVpZ2h0PSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4gIDxwYXRoIGQ9Ik0uODc2IDguMzF2LS42Mmw3IDcuNTNhLjQ1Mi40NTIgMCAwIDEtLjA0Ny42NjQuNTIyLjUyMiAwIDAgMS0uNzA1LS4wNDVsLTctNy41M2EuNDUuNDUgMCAwIDEgMC0uNjE5bDctNy41M2EuNTIyLjUyMiAwIDAgMSAuNzA1LS4wNDQuNDUyLjQ1MiAwIDAgMSAuMDQ3LjY2NGwtNyA3LjUzeiIgZmlsbD0iIzE3ODVGQiIvPjwvc3ZnPg==);
}

.c0 .DayPicker-NavButton--next {
  right: 0;
  background-image: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iOCIgaGVpZ2h0PSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4gIDxwYXRoIGQ9Ik03LjEyNCA4LjMxbC03LTcuNTNBLjQ1Mi40NTIgMCAwIDEgLjE3LjExNmEuNTIyLjUyMiAwIDAgMSAuNzA1LjA0NWw3IDcuNTNhLjQ1LjQ1IDAgMCAxIDAgLjYxOWwtNyA3LjUzYS41MjIuNTIyIDAgMCAxLS43MDUuMDQ0LjQ1Mi40NTIgMCAwIDEtLjA0Ny0uNjY0bDctNy41M3YuNjJ6IiBmaWxsPSIjMTc4NUZCIi8+PC9zdmc+);
}

.c0 .DayPicker-NavButton--interactionDisabled,
.c0 .DayPicker-NavButton--interactionDisabled:hover {
  cursor: not-allowed;
  opacity: 0.3;
}

.c0 .DayPicker-Caption {
  margin-bottom: 8px;
  text-align: center;
}

.c0 .DayPicker-Caption div {
  display: inline-block;
  position: relative;
  z-index: 2;
  overflow: hidden;
  margin: 0;
  padding: 0 3px;
  font: 500 12px/20px 'Roboto';
  background-color: #fff;
}

.c0 .DayPicker-Caption select {
  cursor: pointer;
  position: absolute;
  z-index: 1;
  margin: 0;
  left: 0;
  top: 0;
  opacity: 0;
}

<div
  className="c0"
>
  <div
    className="DayPicker"
    lang="en"
  >
    <div
      className="DayPicker-wrapper"
      onBlur={undefined}
      onFocus={undefined}
      onKeyDown={[Function]}
      tabIndex={-1}
    >
      <div
        className="DayPicker-NavBar"
      >
        <span
          aria-label="Previous Month"
          className="DayPicker-NavButton DayPicker-NavButton--prev DayPicker-NavButton--interactionDisabled"
          onClick={undefined}
          onKeyDown={undefined}
          role="button"
          tabIndex="0"
        />
        <span
          aria-label="Next Month"
          className="DayPicker-NavButton DayPicker-NavButton--next"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex="0"
        />
      </div>
      <div
        className="DayPicker-Months"
      >
        <div
          className="DayPicker-Month"
          role="grid"
        >
          <form
            className="DayPicker-Caption"
          >
            <div>
              September
              <select
                name="month"
                onChange={[Function]}
                value={8}
              >
                <option
                  disabled={true}
                  value={0}
                >
                  January
                </option>
                <option
                  disabled={true}
                  value={1}
                >
                  February
                </option>
                <option
                  disabled={true}
                  value={2}
                >
                  March
                </option>
                <option
                  disabled={true}
                  value={3}
                >
                  April
                </option>
                <option
                  disabled={true}
                  value={4}
                >
                  May
                </option>
                <option
                  disabled={true}
                  value={5}
                >
                  June
                </option>
                <option
                  disabled={true}
                  value={6}
                >
                  July
                </option>
                <option
                  disabled={true}
                  value={7}
                >
                  August
                </option>
                <option
                  disabled={false}
                  value={8}
                >
                  September
                </option>
                <option
                  disabled={false}
                  value={9}
                >
                  October
                </option>
                <option
                  disabled={false}
                  value={10}
                >
                  November
                </option>
                <option
                  disabled={false}
                  value={11}
                >
                  December
                </option>
              </select>
            </div>
            <div>
              2018
              <select
                name="year"
                onChange={[Function]}
                value={2018}
              >
                <option
                  value={2018}
                >
                  2018
                </option>
                <option
                  value={2019}
                >
                  2019
                </option>
                <option
                  value={2020}
                >
                  2020
                </option>
              </select>
            </div>
          </form>
          <div
            className="DayPicker-Weekdays"
            role="rowgroup"
          >
            <div
              className="DayPicker-WeekdaysRow"
              role="row"
            >
              <div
                className="DayPicker-Weekday"
                role="columnheader"
              >
                <abbr
                  title="Monday"
                >
                  Mo
                </abbr>
              </div>
              <div
                className="DayPicker-Weekday"
                role="columnheader"
              >
                <abbr
                  title="Tuesday"
                >
                  Tu
                </abbr>
              </div>
              <div
                className="DayPicker-Weekday"
                role="columnheader"
              >
                <abbr
                  title="Wednesday"
                >
                  We
                </abbr>
              </div>
              <div
                className="DayPicker-Weekday"
                role="columnheader"
              >
                <abbr
                  title="Thursday"
                >
                  Th
                </abbr>
              </div>
              <div
                className="DayPicker-Weekday"
                role="columnheader"
              >
                <abbr
                  title="Friday"
                >
                  Fr
                </abbr>
              </div>
              <div
                className="DayPicker-Weekday"
                role="columnheader"
              >
                <abbr
                  title="Saturday"
                >
                  Sa
                </abbr>
              </div>
              <div
                className="DayPicker-Weekday"
                role="columnheader"
              >
                <abbr
                  title="Sunday"
                >
                  Su
                </abbr>
              </div>
            </div>
          </div>
          <div
            className="DayPicker-Body"
            role="rowgroup"
          >
            <div
              className="DayPicker-Week"
              role="row"
            >
              <div
                aria-disabled={true}
                aria-label="Mon Aug 27 2018"
                aria-selected={false}
                className="DayPicker-Day DayPicker-Day--disabled DayPicker-Day--outside"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                27
              </div>
              <div
                aria-disabled={true}
                aria-label="Tue Aug 28 2018"
                aria-selected={false}
                className="DayPicker-Day DayPicker-Day--disabled DayPicker-Day--outside"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                28
              </div>
              <div
                aria-disabled={true}
                aria-label="Wed Aug 29 2018"
                aria-selected={false}
                className="DayPicker-Day DayPicker-Day--disabled DayPicker-Day--outside"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                29
              </div>
              <div
                aria-disabled={true}
                aria-label="Thu Aug 30 2018"
                aria-selected={false}
                className="DayPicker-Day DayPicker-Day--disabled DayPicker-Day--outside"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                30
              </div>
              <div
                aria-disabled={true}
                aria-label="Fri Aug 31 2018"
                aria-selected={false}
                className="DayPicker-Day DayPicker-Day--disabled DayPicker-Day--outside"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                31
              </div>
              <div
                aria-disabled={false}
                aria-label="Sat Sep 01 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={0}
              >
                1
              </div>
              <div
                aria-disabled={false}
                aria-label="Sun Sep 02 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                2
              </div>
            </div>
            <div
              className="DayPicker-Week"
              role="row"
            >
              <div
                aria-disabled={false}
                aria-label="Mon Sep 03 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                3
              </div>
              <div
                aria-disabled={false}
                aria-label="Tue Sep 04 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                4
              </div>
              <div
                aria-disabled={false}
                aria-label="Wed Sep 05 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                5
              </div>
              <div
                aria-disabled={false}
                aria-label="Thu Sep 06 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                6
              </div>
              <div
                aria-disabled={false}
                aria-label="Fri Sep 07 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                7
              </div>
              <div
                aria-disabled={false}
                aria-label="Sat Sep 08 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                8
              </div>
              <div
                aria-disabled={false}
                aria-label="Sun Sep 09 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                9
              </div>
            </div>
            <div
              className="DayPicker-Week"
              role="row"
            >
              <div
                aria-disabled={false}
                aria-label="Mon Sep 10 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                10
              </div>
              <div
                aria-disabled={false}
                aria-label="Tue Sep 11 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                11
              </div>
              <div
                aria-disabled={false}
                aria-label="Wed Sep 12 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                12
              </div>
              <div
                aria-disabled={false}
                aria-label="Thu Sep 13 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                13
              </div>
              <div
                aria-disabled={false}
                aria-label="Fri Sep 14 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                14
              </div>
              <div
                aria-disabled={false}
                aria-label="Sat Sep 15 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                15
              </div>
              <div
                aria-disabled={false}
                aria-label="Sun Sep 16 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                16
              </div>
            </div>
            <div
              className="DayPicker-Week"
              role="row"
            >
              <div
                aria-disabled={false}
                aria-label="Mon Sep 17 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                17
              </div>
              <div
                aria-disabled={false}
                aria-label="Tue Sep 18 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                18
              </div>
              <div
                aria-disabled={false}
                aria-label="Wed Sep 19 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                19
              </div>
              <div
                aria-disabled={false}
                aria-label="Thu Sep 20 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                20
              </div>
              <div
                aria-disabled={false}
                aria-label="Fri Sep 21 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                21
              </div>
              <div
                aria-disabled={false}
                aria-label="Sat Sep 22 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                22
              </div>
              <div
                aria-disabled={false}
                aria-label="Sun Sep 23 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                23
              </div>
            </div>
            <div
              className="DayPicker-Week"
              role="row"
            >
              <div
                aria-disabled={false}
                aria-label="Mon Sep 24 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                24
              </div>
              <div
                aria-disabled={false}
                aria-label="Tue Sep 25 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                25
              </div>
              <div
                aria-disabled={false}
                aria-label="Wed Sep 26 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                26
              </div>
              <div
                aria-disabled={false}
                aria-label="Thu Sep 27 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                27
              </div>
              <div
                aria-disabled={false}
                aria-label="Fri Sep 28 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                28
              </div>
              <div
                aria-disabled={false}
                aria-label="Sat Sep 29 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                29
              </div>
              <div
                aria-disabled={false}
                aria-label="Sun Sep 30 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                30
              </div>
            </div>
            <div
              className="DayPicker-Week"
              role="row"
            >
              <div
                aria-disabled={true}
                aria-label="Mon Oct 01 2018"
                aria-selected={false}
                className="DayPicker-Day DayPicker-Day--outside"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                1
              </div>
              <div
                aria-disabled={true}
                aria-label="Tue Oct 02 2018"
                aria-selected={false}
                className="DayPicker-Day DayPicker-Day--outside"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                2
              </div>
              <div
                aria-disabled={true}
                aria-label="Wed Oct 03 2018"
                aria-selected={false}
                className="DayPicker-Day DayPicker-Day--outside"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                3
              </div>
              <div
                aria-disabled={true}
                aria-label="Thu Oct 04 2018"
                aria-selected={false}
                className="DayPicker-Day DayPicker-Day--outside"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                4
              </div>
              <div
                aria-disabled={true}
                aria-label="Fri Oct 05 2018"
                aria-selected={false}
                className="DayPicker-Day DayPicker-Day--outside"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                5
              </div>
              <div
                aria-disabled={true}
                aria-label="Sat Oct 06 2018"
                aria-selected={false}
                className="DayPicker-Day DayPicker-Day--outside"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                6
              </div>
              <div
                aria-disabled={true}
                aria-label="Sun Oct 07 2018"
                aria-selected={false}
                className="DayPicker-Day DayPicker-Day--outside"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={[Function]}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                7
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
